### ISSUE TITLE
firstPublisher is not equivalent but subproperty of schema:publisher

### ISSUE THREAD

**Issue by RubenVerborgh** [dbpedia/mappings-tracker#90]
Currently `dbo:firstPublisher` is listed as `owl:equivalentProperty` of `schema:publisher`. However, since `schema:publisher` also includes people in its range, and `dbo:firstPublisher` assumes companies, this leads to inconsistencies.

**Comment by jimkont** [comment]
Thanks for the report @RubenVerborgh 
For me, the best option would be to change the range of `dbo:firstPublisher` to `dbo:Agent`

**Comment by RubenVerborgh** [comment]
It depends:
- if all values of `dbo:firstPublisher` are in fact companies, we might like to keep the range
- are _first_ publisher and _publisher_ really equivalent (i.e., why “first”)?

**Comment by jimkont** [comment]
OK, let's analyze a bit. This is not a frequenty used property in DBpedia, 
http://mappings.dbpedia.org/index.php/Special:WhatLinksHere/OntologyProperty:FirstPublisher

I think it is used to model the original publisher of a work.
[Checking on the Dutch DBpedia](http://nl.dbpedia.org/sparql?default-graph-uri=http%3A%2F%2Fnl.dbpedia.org&query=select+*+where+%7B%3Fs+%3Chttp%3A%2F%2Fdbpedia.org%2Fontology%2FfirstPublisher%3E+%3Fo%7D+LIMIT+100&format=text%2Fhtml&timeout=30000&debug=on)  I think that there are a few Persons there as well but my Dutch are not very good :-) can you double check?

**Comment by RubenVerborgh** [comment]
> Checking on the Dutch DBpedia I think that there are a few Persons there as well

Indeed, but it is an edge case. E.g., http://nl.dbpedia.org/page/De_iure_belli_ac_pacis is published by http://nl.dbpedia.org/page/Nicholas_Buon, and Buon is a person. However, it seems to be an identifier problem here: I'm pretty sure they actually mean “the publishing company owned by Buon”.

But in general, let's indeed make `dbo:Agent` the range of `dbo:firstPublisher`, then it works in any case.
However, I'd still argue to also make it a subproperty of `schema:publisher`, which does not have the semantics of “original” publisher.

> This is not a frequenty used property in DBpedia

BTW, interesting backstory of on how I found this:
I reasoned over my own dataset with the DBpedia ontology in order to [publish my own data](https://data.verborgh.org/ruben), and I was surprised to see that I was a company. Turns out that I publish blog posts, so DBpedia made me `dbo:firstPublisher`, and, consequently, a company.

**Comment by jimkont** [comment]
Actually, looking at `dbo:publisher`
We should both change the range to `dbo:Agent` and remove the equivalence to `schema:publisher`

**Comment by RubenVerborgh** [comment]
:+1:, but subproperty would also be nice and correct.

**Comment by jimkont** [comment]
even better, it's fixed now and will be updated on the next release
http://mappings.dbpedia.org/index.php/OntologyProperty:FirstPublisher

**Comment by VladimirAlexiev** [comment]
@rubenverborgh if you infer any types from Dbpedia domains\ranges, you're in for nasty surprises. Eg
``` 
| mother = [[Victoria]], queen of [[England]]
```
Will infer that England is a person

**Comment by barrynl** [comment]
Hi, I think this (old) issue can be closed: https://dbpedia.org/ontology/firstPublisher
